src-backbone/app/js/utils/helpers.js

```diff
@@ -0,0 +1,35 @@
+var Config = require('utils/config');
+
+module.exports = {
+
+    goto_default_logged_in: function() {
```

Comment:
Why are method names here snake case? I'm not sure if it's different for helper methods but isn't convention to use camel case?